src/dev/test_generator/generate/strategies/hypothesis.py

```
@@ -0,0 +1,131 @@
+                return fix_generated_code(raw)
+        return None
+
+    # ------------------------------------------------------------------
+    # Private helpers
+    # ------------------------------------------------------------------
+
+    def _run_hypothesis_write(self, args: str) -> Optional[str]:
+        env = self._build_env()
+        cmd = f"{self._hypothesis_bin()} write {args}"
+        try:
+            result = subprocess.run(
```

Ruff (0.15.7)

[error] `subprocess` call with `shell=True` identified, security issue

(S602)